src/components/AlertMessage.jsx

```diff
@@ -30,4 +31,10 @@ class AlertMessage extends React.Component {
 	}
 }
 
+AlertMessage.propTypes = {
+	type: PropTypes.string.isRequired,
+	children: PropTypes.string,
```

Comment:
You could send many things as children, numbers, react elements, arrays etc. Is there a proptype that encapsulates all of those?